.travis.yml

```diff
@@ -1,37 +1,40 @@
 sudo: false
 language: python
 python:
+- '2.6'
+- '2.7'
+- '3.3'
+- '3.4'
 - '3.5'
+- nightly
+
+cache: pip
 install: pip install -U tox setuptools_scm
 env:
   matrix:
-  - TESTENV=py26-pytest26
```

Comment:
it gives more correct results from travis
i#d like to see a tox plugin to work directly on top of that